### ISSUE TITLE
[BUG] Tabby Logs Prompts  When `prompt: False`

### ISSUE THREAD

**Issue by strikeoncmputrz** [theroyallab/tabbyAPI#176]
### OS

Linux

### GPU Library

CUDA 12.x

### Python version

3.12

### Describe the bug

 When I send a chat completion request to tabby, the prompt is being logged to stdout.  If I understand the parameters in config.yml, this should not be occurring.  I copy and pasted the default logging config directly from config_example.yml. Is this operator error or a bug? 

### Reproduction steps

 I have the following logging configuration in config.yml

```
 # Options for logging                                                                                                                                                    
 logging:                                                                                                                                                                 
   # Enable prompt logging (default: False)                                                                                                                               
   prompt: False                                                                                                                                                          
                                                                                                                                                                          
   # Enable generation parameter logging (default: False)                                                                                                                 
   generation_params: False                                                                                                                                               
                                                                                                                                                                          
   # Enable request logging (default: False)                                                                                                                              
   # NOTE: Only use this for debugging!                                                                                                                                   
   requests: False 
   ```
   
 Request:

   ```
   #!/bin/bash
curl -X POST "http://llama.lan:5000/v1/chat/completions" \
         -H "Content-Type: application/json" \
              -H "Authorization: Bearer REDACTED" \
              -d '{
           "model": "LoneStriker_Hermes-3-Llama-3.1-70B-6.0bpw-h6-exl2_main",
           "messages": [
                          {"role": "user", "content": "Hello! What can you do?"}

           ],
                      "temperature": 0.7

          }'

   ```
   
   
 TabbyAPI stdout:
   
  ```
INFO:     Finished chat completion request cbb166dcb0af492da5b5652e1bcdf7b0
INFO:     192.168.10.107:56810 - "POST /v1/chat/completions HTTP/1.1" 200
<|im_start|>user
Hello! What can you do?<|im_end|>
<|im_start|>assistant

[]
INFO:     Metrics (ID: 5be2aea5df42494bbb395ed8051481c4): 242 tokens generated in 35.26 seconds (Queue: 0.0 s, Process: 0 cached tokens and 16 new tokens at 36.37 T/s, 
Generate: 6.95 T/s, Context: 16 tokens) 
INFO:     Finished chat completion request 5be2aea5df42494bbb395ed8051481c4
```
   

### Expected behavior

TabbyAPI does not log prompts when `prompt: False` is set in config.yml

### Logs

INFO:     Finished chat completion request cbb166dcb0af492da5b5652e1bcdf7b0
INFO:     192.168.10.107:56810 - "POST /v1/chat/completions HTTP/1.1" 200
<|im_start|>user
Hello! What can you do?<|im_end|>
<|im_start|>assistant

[]
INFO:     Metrics (ID: 5be2aea5df42494bbb395ed8051481c4): 242 tokens generated in 35.26 seconds (Queue: 0.0 s, Process: 0 cached tokens and 16 new tokens at 36.37 T/s, 
Generate: 6.95 T/s, Context: 16 tokens) 
INFO:     Finished chat completion request 5be2aea5df42494bbb395ed8051481c4

### Additional context

_No response_

### Acknowledgements

- [X] I have looked for similar issues before submitting this one.
- [X] I have read the disclaimer, and this issue is related to a code bug. If I have a question, I will use the Discord server.
- [X] I understand that the developers have lives and my issue will be answered when possible.
- [X] I understand the developers of this program are human, and I will ask my questions politely.

**Comment by gittb** [comment]
Addressed with pr #174

**Comment by bdashore3** [comment]
Merged #174, closing issue.
